sktime/classification/deep_learning/rnn.py

```diff
@@ -12,7 +12,7 @@
 from sktime.networks.rnn import RNNNetwork
 from sktime.utils.validation._dependencies import _check_dl_dependencies
 
-
+#todo: 0.26.0 - parameter num_epochs is deprecated and n_epoches is used instead
```

Comment:
Yes, exactly that is what was meant!

Though, this should be instructions for the release manager, so it should be very explicit:

"please remove num_epochs parameter and related logic, because ... (then what you wrote)"